build-tools/Xamarin.Android.Tools.BootstrapTasks/Xamarin.Android.Tools.BootstrapTasks/CheckApiCompatibility.cs

```diff
@@ -142,44 +142,41 @@ void ValidateApiCompat (string contractPath, bool validateAgainstReference)
 					File.Copy (implementationAssembly, Path.Combine (targetImplementationPathDirectory, assemblyToValidate), true);
 				}
 
-				using (var genApiProcess = new Process ()) {
+				for (int i = 0; i < 3; i++) {
```

Comment:
PR #4174 added support to re-execute `Microsoft.DotNet.ApiCompat.exe` if/when mono crashes.  What are these more recent changes for?  I don't see a rationale for these changes within the PR message.

Reply:
I wanted to created a new Process object to re-try instead of using the one already created. For some reason I was getting some odd behavior when re-running the process from the same Process object after the crash. When I was reading online, the recommendation was to dispose the object and use a new one.